code/LINDA/LINDA_turf_tile.dm

```diff
@@ -211,7 +220,14 @@
 
 	if(planet_atmos) //share our air with the "atmosphere" "above" the turf
 		var/datum/gas_mixture/G = new
-		G.copy_from_turf(src)
```

Comment:
Why not use `copy_from_turf` here?

Reply:
so, here's `copy_from_turf`:

```
/datum/gas_mixture/copy_from_turf(turf/model)
	oxygen = model.oxygen
	carbon_dioxide = model.carbon_dioxide
	nitrogen = model.nitrogen
	toxins = model.toxins

	//acounts for changes in temperature
	var/turf/model_parent = model.parent_type
	if(model.temperature != initial(model.temperature) || model.temperature != initial(model_parent.temperature))
		temperature = model.temperature

	return 1
```

The problem with this is the temperature checks. the vars for gases on a turf are *NEVER* altered under any circumstances; they're used for turf initialization with air, so we have an easy, non string way to handle what gases a turf has by default.

Temperature is a double whammy---it' used simultaneously for initialization temperature...anddd it's involved in atmos superconduction and is modified at runtime.

Having this proc run ultimately means that if the turf winds up with a temperature of something like 1,000,000 or something, it'll copy that to the air above it.